modules/ai/assets/js/editor/components/usage-limit-alert.js

```diff
@@ -13,7 +13,13 @@ const CREDITS_80_USAGE_TITLE = getUsageTitle( '80%' );
 const CREDITS_75_USAGE_TITLE = getUsageTitle( '75%' );
 
 const DESCRIPTION_SUBSCRIPTION = __( 'Get maximum access.', 'elementor' );
-const DESCRIPTION_NO_SUBSCRIPTION = __( 'Upgrade now to keep using this feature. You still have credits for other AI features (Text, Code, Images, Containers, etc.)', 'elementor' );
+const FEATURES = [ 'Text', 'Images', 'Code', 'Containers' ];
```

Comment:
There is nothing that alert the next dev that will update this list that need to update all UsageMessages uses. 
lets save the values in an enum

Reply:
Check again @naorsabag